src/core/middleware.py

```diff
@@ -22,48 +24,50 @@ def process_request(self, request):
         # if request doesn't have all expected headers, strip auth
         if CGROUPS not in request.META:
             # no auth or invalid auth request, return immediately
-            LOG.debug('header %r not found, refusing auth', CGROUPS)
+            LOG.debug("header %r not found, refusing auth", CGROUPS)
```

Comment:
did black add double quotes to everything?

Reply:
yup